cleanup

```
@@ -2,12 +2,21 @@
 
 log()   { echo ">>> $1" ; }
 
-[[ -d manifest-templates ]] || { echo >&2 "Please run this script from within the kubernetes folder"; exit 1; }
+[[ -d manifest-templates ]] || { echo >&2 "Please run this script from within the root folder"; exit 1; }
+
+log "comparing remote and local Gemfile.lock to see if the local development image needs to be rebuilt"
+docker run --rm -v $VELUM_DIR:/srv/velum sles12/velum:development diff /var/lib/velum/Gemfile.lock /srv/velum/Gemfile.lock &> /dev/null
```

I think we should only run this if the `sles12/velum:development` is present, otherwise it will fail. We might just ignore the error... but wanted to point it out.